src/applications/vaos/tests/new-appointment/components/TypeOfCarePage.unit.spec.jsx

```diff
@@ -374,4 +375,49 @@ describe('VAOS <TypeOfCarePage>', () => {
       ),
     );
   });
+
+  describe('using VAOS service', () => {
+    it('should open facility type page when CC eligible and has a supported parent site', async () => {
```

Comment:
@LeKeve @vbahinwillit I think we could update the parent sites call in this test with your changes, and cover both our tickets (mostly) in this one test. Or if you have this test copied in your work, I can update that with our supported sites changes.

Reply:
I think we can update this test with our parent facilities request once it's merged since we have some additional changes to make in our PR.

Reply:
Sounds good. I decided to add a second test to cover when there aren't any supported sites, so probably can update that one, too.